app/policies/cohort_report/facility_group_policy.rb

```diff
@@ -1,4 +1,16 @@
 class CohortReport::FacilityGroupPolicy < ApplicationPolicy
+  def patient_list?
+    user_has_any_permissions?(
+      [:download_patient_line_list, nil],
+      [:download_patient_line_list, record.organization],
+      [:download_patient_line_list, record]
+    )
+  end
```

Comment:
This was an attempt to add the pundit check for patient_list rights for the Facility Group.  This doesn't seem right tho.

![image](https://user-images.githubusercontent.com/69/93266833-4e489980-f770-11ea-9449-ea501ada813a.png)

Reply:
This looks accurate to me.